DOCKER.md

```diff
@@ -12,7 +12,7 @@ For the following commands sudo might be needed depending on whether your user i
 
 To build the image :
 
-    ./docker-build
+    docker build -t fitiotlab/iot-lab-gateway .
```

Comment:
I'm OK with this, I think we'd better remove the fitiotlab prefix. Even if we might need it if we're pushing the built image to https://hub.docker.com/u/fitiotlab/ (an account we created during the sprint code), it's not needed to build and run locally

Reply:
FYI, probable typo L64 in DOCKER.md    `./docker-run -v` => `docker-run -d`

Reply:
I agree. pushed in PR#1 on your fork